### PR TITLE
[4.0] RavenDB-10812 introduced LastAccessTime that is bumped when database …

### PR DESCRIPTION
**Pull request by ppekrol**
…is being accessed by DatabasesLandlord (e.g. on request)

### CODE REVIEW

File: src/Raven.Server/Documents/DocumentDatabase.cs [hunk 1]
```diff
@@ -138,6 +139,8 @@ public DocumentDatabase(string name, RavenConfiguration configuration, ServerSto
 
         public DateTime LastIdleTime => new DateTime(_lastIdleTicks);
 
+        public DateTime LastAccessTime;
```
Comment (ayende) [inline src/Raven.Server/Documents/DocumentDatabase.cs:142]: Where is this actually being used?

Comment (ppekrol) [reply src/Raven.Server/Documents/DocumentDatabase.cs:142]: To calculate LastWork?

Comment (ayende) [reply src/Raven.Server/Documents/DocumentDatabase.cs:142]: But I don't see it being used now?

File: src/Raven.Server/Documents/DatabasesLandlord.cs [hunk 2]
```diff
@@ -641,7 +644,7 @@ public DateTime LastWork(DocumentDatabase resource)
             var envs = resource.GetAllStoragesEnvironment();
 
             long dbSize = 0;
-            var maxLastWork = DateTime.MinValue;
+            var maxLastWork = resource.LastAccessTime;
```
Comment (ppekrol) [inline src/Raven.Server/Documents/DatabasesLandlord.cs:647]: @ayende It is used here

**Pull request merged**